framework/platform/bpl/linux/prplmesh_platform_db

```diff
@@ -2,3 +2,10 @@ management_mode=Multi-AP-Controller-and-Agent
 operating_mode=Gateway
 stop_on_failure_attempts=1
 certification_mode=1
+enable=1
```

Comment:
Here you should add only the mandatory arguments and mention that any argument which is not specified is using default value